numba/cpython/mathimpl.py

```diff
@@ -176,6 +176,7 @@ def float_impl(context, builder, sig, args):
 exp_impl = unary_math_intr(math.exp, 'llvm.exp')
 log_impl = unary_math_intr(math.log, 'llvm.log')
 log10_impl = unary_math_intr(math.log10, 'llvm.log10')
+log2_impl = unary_math_intr(math.log2, 'llvm.log2')
```

Comment:
Technically this implementation should raise `ValueError('math domain error')` for input values that are zero or negative, however, what is present is no different to e.g. `log10` above which would suffer from the same issue.

Reply:
I can tackle this in a follow-up PR if needed.